xpholder/commands/mod/editOtherCharacter.js

```diff
@@ -54,7 +53,9 @@ module.exports = {
         INITALIZATIONS
         --------------
         */
-        const player = interaction.options.getUser("player");
+        const player = await guild.members.fetch(
```

Comment:
You want to move your player declaration ( `const player = ...` ) below where guilds are declared ( `const guild = ...`), otherwise the player has no context of what a guild is, and the code will either have an unpredicted behavior because of a higher scope guild declaration ( which is still bad) or outright fail because no such variables exists.